source/module_base/test/spherical_bessel_transformer_test.cpp

```diff
@@ -161,8 +161,7 @@ TEST_F(SphericalBesselTransformTest, RadrfftImplicitExponent)
     double dk = PI / rcut;
     double pref = std::sqrt(2. / PI) * 48.;
```

Comment:
const double pref?

Reply:
a const qualifer has been added in the latest commit